src/scenarios/shared/startup.py

```diff
@@ -16,12 +16,14 @@ class StartupWrapper(object):
     Wraps startup.exe, building it if necessary.
     '''
     def __init__(self):
-        payload_startup = os.path.join(helixpayload(), 'Startup')
-        workitem_startup = os.path.join(helixworkitempayload(), 'Startup')
-        if os.path.exists(payload_startup):
-            self._setstartuppath(payload_startup)
-        elif os.path.exists(workitem_startup):
-            self._setstartuppath(workitem_startup)
+        if helixpayload():
+            payload_startup = os.path.join(helixpayload(), 'Startup')
+            if os.path.exists(payload_startup):
+                self._setstartuppath(payload_startup)
+        elif helixworkitempayload():
+            workitem_startup = os.path.join(helixworkitempayload(), 'Startup')
+            if os.path.exists(workitem_startup):
+                self._setstartuppath(workitem_startup)
```

Comment:
Aha. It only breaks locally, I guess, not in CI. I thought I tested it but I must not have. Sorry.

Either way, this change reintroduces the logic problem I was fixing. This will always take the first branch in Helix, because the variable is always set. 

```suggestion
        if helixpayload() and os.path.exists(os.path.join(helixpayload(), 'Startup'):
            self._setstartuppath(os.path.join(helixpayload(), 'Startup')
        elif helixworkitempayload() and os.path.exists(os.path.join(helixworkitempayload(), 'Startup')):
            self._setstartuppath(os.path.join(helixworkitempayload(), 'Startup'))
```

Reply:
Oh you're right ! I'll commit the suggestion right away.

Reply:
I only know this because I wrote the bug the first time. :)